### ISSUE TITLE
[Feature Request] Read-Only Status In Status Bar For Locked Files

### ISSUE THREAD

**Issue by alienfxfiend** [notepad-plus-plus/notepad-plus-plus#5573]
### Description of the Issue

Read-Only displayed in the Status Bar would be very handy, currently it's available in EmEditor Pro but I would love to see it in Notepad++ 'cause I use à laptop & have to protect my files in case of accidental overwrite. Of course only if there's room in the Status Bar for this, otherwise I can understand… I use the shortcut Alt+Ro to make a document Read-Only (changed the Shortcut through Mapper) & also made a Context Menu entry for it, but I have to view them in Doc Switcher to know if they're Read-Only —I think having a Status Bar notification would be excellent\ handy.

### Steps to Reproduce the Issue

1. 
2. 
3.

**Comment by sasumner** [comment]
On every editing tab there is a little floppy diskette (how quaint in 2019!).  It is grey color (instead of the normal blue color) when a file is read-only.  It is a bit subtle to notice.

There is no possibility of "accidental overwrite".  If you try typing into a read-only tab buffer, nothing happens!

**Comment by alienfxfiend** [comment]
> 
> 
> On every editing tab there is a little floppy diskette (how quaint in 2019!). It is grey color (instead of the normal blue color) when a file is read-only. It is a bit subtle to notice.
> 
> There is no possibility of "accidental overwrite". If you try typing into a read-only tab buffer, nothing happens!

Yes I know Read-Only prevents editing but I fall asleep on the laptop —& it would be more visible to have the Read-Only status also in the Status Bar…

**Comment by sasumner** [comment]
>but add this supplementarily

Sure...as a suggestion goes, it's fine. :)

I was just pointing out some things that I couldn't tell if you already knew about.

**Comment by alankilborn** [comment]
>but I fall asleep on the laptop

User probably should not suggest new Notepad++ features to cure his sleeping problems. :-)

With the "lock" icon on the tab of a readonly file when "Alternate icons" are selected in the Preferences, this is probably never going to be sanctioned as a change, so recommend CLOSURE of this item.

**Comment by shoulders** [comment]
@alienfxfiend can you close this issue. thanks
